packages/create-plugin/src/commands/generate/prettify-files.ts

```diff
@@ -4,13 +4,13 @@ import fs from 'node:fs';
 
```

Comment:
nit: move this file out of the `generate` folder since it is now used in more than the generate command.